encoding/kzg/prover/compute_proof.go

```diff
@@ -0,0 +1,206 @@
+package prover
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/encoding/fft"
+	"github.com/Layr-Labs/eigenda/encoding/kzg"
+	"github.com/Layr-Labs/eigenda/encoding/utils/toeplitz"
+	"github.com/consensys/gnark-crypto/ecc"
+	"github.com/consensys/gnark-crypto/ecc/bn254"
+	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
+)
+
+type CpuComputer struct {
```

Comment:
nit: Rename to CPUComputer or CPUProofComputer

Reply:
+1 to CPUProofComputer

Reply:
I dont think we should add cpu into the name, since in the future, it could have been a gpu. Either cpu or gpu, I don't think it makes sense to have both